rest/route/repo_test.go

```diff
@@ -307,190 +315,4 @@ func TestPatchHandlersWithRestricted(t *testing.T) {
 	assert.Len(t, allProjectsScope.Resources, 2)
 	assert.Contains(t, allProjectsScope.Resources, "branch1")
 	assert.Contains(t, allProjectsScope.Resources, "branch2")
-
```

Comment:
This test doesn't play nice with the new setup and I wasn't successful in figuring out how to re-write it.